### PR TITLE
QF-2037-Expire_employer_requests

### PR DESCRIPTION
**Pull request by karen-hanna**

### CODE REVIEW

File: src/SFA.DAS.RequestApprenticeTraining.Jobs/Functions/SampleTimerFunction.cs [hunk 1]
```diff
@@ -26,7 +26,7 @@ public async Task SampleTimer(
             try
             {
                 _log.LogInformation($"SampleTimer has started");
-                var employerRequest = await _api.GetEmployerRequest(new Guid("EC4CC0DA-786E-4339-D40A-08DCB6FBD772"));
+                var employerRequest = await _api.GetEmployerRequest(new Guid("1B02548D-2108-461A-A0B2-E9419B4E0A69"));
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.RequestApprenticeTraining.Jobs/Functions/SampleTimerFunction.cs]: The SampleTimerFunction can be deleted now

File: src/SFA.DAS.RequestApprenticeTraining.Infrastructure/Configuration/FunctionsOptions.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.RequestApprenticeTraining.Infrastructure.Configuration
+{
+    public class FunctionsOptions
+    {
+        public string SampleTimerSchedule { get; set; }
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.RequestApprenticeTraining.Infrastructure/Configuration/FunctionsOptions.cs]: The SampleTimerFunction can be deleted now

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.RequestApprenticeTraining.Infrastructure/Configuration/FunctionsOptions.cs (hunk 2 above)
Comment (karen-hanna) [reply src/SFA.DAS.RequestApprenticeTraining.Infrastructure/Configuration/FunctionsOptions.cs]: removed

File: src/SFA.DAS.RequestApprenticeTraining.Jobs/Functions/SampleTimerFunction.cs (hunk 1 above)
Comment (karen-hanna) [reply src/SFA.DAS.RequestApprenticeTraining.Jobs/Functions/SampleTimerFunction.cs]: removed

**Review by ChrisJWoodcock — DISMISSED** [review]

**Review by ChrisJWoodcock — APPROVED** [review]

**Review by ChrisJWoodcock — APPROVED** [review]

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**